### PR TITLE
Log fixes and improvements

### PR DESCRIPTION
**Pull request by juliasaveliff**
remember when ben sat on an epic log?

### CODE REVIEW

File: rackcity/utils/log_utils.py [hunk 1]
```diff
@@ -163,29 +167,19 @@ def log_power_action(user, power_action, related_asset):
     log.save()
 
 
-def log_network_action(user, action, asset_0, asset_1):
+def log_network_action(user, related_asset):
```
Comment (juliasaveliff) [inline rackcity/utils/log_utils.py]: there was no clear point in the creation/deletion of network connections where it made sense to call this log. so now network connection actions will be logged only via the asset they were done from

File: src/components/logs/logs.scss [hunk 2]
```diff
@@ -34,4 +34,7 @@ form {
   -moz-box-sizing: content-box;
   -webkit-box-sizing: content-box;
   box-sizing: content-box;
+}
+.log-block {
+  overflow: scroll;
```
Comment (clairecfu) [inline src/components/logs/logs.scss]: wooo

File: rackcity/utils/log_utils.py (hunk 1 above)
Comment (clairecfu) [reply rackcity/utils/log_utils.py]: do we need to have logging for bulk import? / will this support network bulk import?

Comment (juliasaveliff) [reply rackcity/utils/log_utils.py]: going to go back and perfect the bulk import logging and errors in one swoop later!

**Pull request merged**